k9mail/src/main/java/com/fsck/k9/message/MessageBuilder.java

```diff
@@ -42,6 +42,8 @@
     private final Context context;
     private final MessageIdGenerator messageIdGenerator;
     private final BoundaryGenerator boundaryGenerator;
+    private static final String highPriorityTag = "X-Priority";
```

Comment:
this is duplicate now. I suggest moving it into `MimeHeader`, we already have a bunch of other headers there